docker-compose.prod.yaml

```
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  web:
+    build:
+      target: production
+  bot:
+    build:
+      target: production
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)